### PR TITLE
Event storage tests

### PR DESCRIPTION
**Pull request by AlexanderLitus**

### CODE REVIEW

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL.

**Comment by alexander-yevsyukov** [comment]
Conflicts emerged. Please resolve them.

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL.

File: core-java/src/main/java/org/spine3/util/Methods.java [hunk 1]
```diff
@@ -63,8 +63,8 @@ public static String getFullMethodName(Method method) {
      * @throws ClassCastException if the first parameter isn't a class implementing {@link Message}
```
Comment (alexander-yevsyukov) [inline core-java/src/main/java/org/spine3/util/Methods.java]: Please remove the link to `Class` from the parameter description. Just `the class` would be enough.

**Comment by alexander-yevsyukov** [comment]
Other than the comment above, LGTM.

**Pull request merged**